sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/StateSchemaCompatibilityChecker.scala

```diff
@@ -153,6 +155,7 @@ class StateSchemaCompatibilityChecker(
         valueSchema.toString)
     } else {
       logInfo("Detected schema change which is compatible. Allowing to put rows.")
+      true
```

Comment:
This comes again. Do we define this to be the case of "schema being evolved"? I don't think we have any support of schema evolution at all.

cc. @anishshri-db Have you happen to know about this? If then do we have some plan for this?